components/lifesensor_macro/include/macro_queue.h

```diff
@@ -0,0 +1,64 @@
+#ifndef __MACRO_QUEUE__
+#define __MACRO_QUEUE__
+
+#include <freertos/queue.h>
+
+
+/**
+ * struct lifesensor_queue_t - generic queue struct
+ * @tcb:	Holds task related (internal) data.
+ * @handle: Provides a reference to the queue.
+ * @buffer: Stores the address of the queues' buffer.
```

Comment:
```suggestion
 * @buffer: Points to the subsequent queue buffer.
```
Does not store anything